packages/component/src/Composer.js

```diff
@@ -573,6 +576,7 @@ Composer.defaultProps = {
   sendTypingIndicator: false,
   styleOptions: {},
   styleSet: undefined,
+  suggestedActionsAccessKey: 'A a Å å',
```

Comment:
Can you explain this for me?

Reply:
Haha. I am also confused until I use Safari on Mac.

IIRC, when you press <kbd>ALT</kbd> + <kbd>SHIFT</kbd> + <kbd>A</kbd>, you will receive few things in the `HTMLEvent` object:

- `keyCode = 65` (deprecated in favor of `code`, I believe)
- `code = "KeyA"` (this don't honor keyboard layout. On a QWERTY keyboard, press <kbd>Y</kbd> -> `"KeyY"`. On a QWERTZ keyboard, press <kbd>Z</kbd> -> `"KeyY"` too)
- `key = "a"` (this honor keyboard layout)

So, `code` is useless. `keyCode`, hmmm, first, deprecated, second, I can't easily convert `65` -> `A`. I need `A` because I need to build the accessibility text: "Press ALT + SHIFT + A for suggested actions".

So, we down to `key`, which is a smaller nightmare.

Then, there are few de facto `accesskey` modifiers.

- Windows
   - <kbd>ALT</kbd> + <kbd>A</kbd>, `key = "a"`. We don't want to capture this one because some browsers ignore it.
   - <kbd>ALT</kbd> + <kbd>SHIFT</kbd> + <kbd>A</kbd>, `key = "a"` (or `"A"`, I don't remember)
- Mac
   - <kbd>CTRL</kbd> + <kbd>OPTION</kbd> + <kbd>A</kbd>, `key = "å"`
   - <kbd>CTRL</kbd> + <kbd>OPTION</kbd> + <kbd>SHIFT</kbd> + <kbd>A</kbd>, `key = "Å"`

On Mac, it's `å` but not an `a`, because <kbd>OPTION</kbd> + <kbd>A</kbd> will yield `å`. So, <kbd>CTRL</kbd> + <kbd>OPTION</kbd> + <kbd>A</kbd> will yield `å`.

HTML `accesskey` officially support 1+ keys, separated by spaces (because <kbd>SPACEBAR</kbd> is not allowed as an access key).

So, we are allowing 4 variants of A. And taking the first one for screen reader text.

I might be able to take out the first one. But since it doesn't hurt, and it is nice for screen reader text, I just leave it there.

(I learn new thing every day.)